docs/linking/unlinking.puml

```diff
@@ -32,18 +32,18 @@ deactivate PISP
 ' TODO: Do we include the ALS lookup here to figure out which Auth service is
 ' responsible for the given Consent?
 
-Switch -> Auth ++:  ""DELETE /consents/123""
+Switch -> Auth ++:  ""POST /consents/123/revoke""
 Auth --> Switch: ""202 Accepted""
 deactivate Switch
 
 Auth -> Auth: Delete the record of the consent.
 
-Auth -> Switch ++: ""DELETE /consents/123""
+Auth -> Switch ++: ""PATCH /consents/123 \n{ consent: {\n  id: 1234,\n  status: REVOKED,\n  revokedAt: `Date`}\n}""
```

Comment:
A few things:
- why the nested `consent`?
- `id` and `status` should be strings